web-app/js/plugin/BoxPlot.js

```diff
@@ -370,5 +370,43 @@ BoxPlotView.prototype.submit_job = function (form) {
 
 }
 
+BoxPlotView.prototype.clear_high_dimensional_input = function (div) {
+    RmodulesView.prototype.clear_high_dimensional_input.call(this, div);
```

Comment:
I don't think so. The function is defined in the prototype. RmodulesView is not an object, but the "super class" of BoxPlotView.